Dockerfile

```diff
@@ -18,4 +15,4 @@ COPY ps-entry.sh /entrypoint.sh
 ENTRYPOINT ["/entrypoint.sh"]
 
 EXPOSE 3306
-CMD ["mysqld"]
+CMD ["mysqld","--user=mysql"]
```

Comment:
Great. Do we think this rules out a Percona-Server change for the build template of my.cnf?